ui/media/js/main.js

```diff
@@ -1378,6 +1445,10 @@ function permutePrompts(promptBase, promptMatrix) {
     return prompts
 }
 
+function permutePromptsNumber(promptMatrix) { // this should calculate how many different prompts can be made with the prompt matrix
```

Comment:
Is this function necessary? How about just calling `permuteNumber(promptMatrix)` in line 1419 directly?

Reply:
Wanted to keep the structure sort of the same to make it easier to follow, but applied suggestion in latest commit.